src/app/item-page/edit-item-page/edit-item-page.component.html

```diff
@@ -4,7 +4,7 @@
             <h2 class="border-bottom">{{'item.edit.head' | translate}}</h2>
             <div class="pt-2">
                 <ul class="nav nav-tabs justify-content-start">
-                    <li *ngFor="let page of pages" class="nav-item">
+                    <li *ngFor="let page of pages" class="nav-item" [attr.aria-selected]="page.page === currentPage">
```

Comment:
I think this looks good!  However, I just realized that the recommendation for tabs is to set the roles "tablist" and "tab".  See the docs at https://www.w3.org/WAI/ARIA/apg/patterns/tabs/

So, I think we need to add two more small things here.  
* First, on the `<ul>` just above this we should have `<ul role="tablist"..`
* Seconding, on this `<li>` we should add the `<li role="tab"..`

Once those are in place, I think this aligns better with the recommendations in the docs above.